nose/case.py

```diff
@@ -357,7 +359,7 @@ def __str__(self):
         name = "%s.%s.%s" % (self.cls.__module__,
                              self.cls.__name__,
                              name)
-        if arg:
+        if not arg == '()':
```

Comment:
No, it's completely intended. `'()'` is the repr of an empty tuple, which being a non-empty string is truthy, whereas before, the empty tuple is falsey. I was maintaining the previous representation of functions which don't have args. This is addressed in the commit message.